packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -497,10 +498,17 @@ export class ContainerRuntime extends EventEmitter
             await readAndParse<[string, string[]][]>(context.storage, chunkId) :
             readAndParseFromBlobs<[string, string[]][]>(context.baseSnapshot.blobs, chunkId) : [];
 
+        const blobId = context.baseSnapshot?.blobs[blobsBlobName];
+        const blobsBlob = blobId
```

Comment:
If possible, I'd not await and pass promise, such that blocking wait can be done only when it's needed.
Even better would be to read from storage only when it's needed. It feels like this info is needed only when summarizing (at least right now) - can we do it only then?

Reply:
If we pass a promise, I don't think we can await it anywhere but here or in BlobManager.snapshot(), which would require making ContainerRuntime.createSummary() async (I assume we don't want this, but maybe it's fine).